NeoCmd/Program.cs

```diff
@@ -196,6 +198,7 @@ protected override void OnFrameExit()
 		private static Lua lua = new Lua(); // create lua script compiler
 		private static LuaGlobal global;
 		private static ILuaDebug debugEngine = LuaStackTraceDebugger.Default;
+		private static bool ClrEnabled = true;

```

Comment:
Please, can refactor to `clrEnabled`

Reply:
class members starting with the lower case...
a) Why?
b) Fixed as requested in 08a5ff2cf6998cbd2131c649f34552d456c7ad69